src/gui/tray/ActivityItem.qml

```diff
@@ -41,17 +41,36 @@ MouseArea {
                                      : message
         Accessible.onPressAction: activityMouseArea.clicked()   
         
-        Image {
-            id: activityIcon
+        Item {
             Layout.alignment: Qt.AlignVCenter | Qt.AlignHCenter
-            Layout.leftMargin: 20
-            Layout.preferredWidth: shareButton.icon.width
-            Layout.preferredHeight: shareButton.icon.height
-            verticalAlignment: Qt.AlignCenter
-            cache: true
-            source: icon
-            sourceSize.height: 64
-            sourceSize.width: 64
+            Layout.leftMargin: 7
+            Layout.preferredWidth: shareButton.icon.width * 2
+            Layout.preferredHeight: shareButton.icon.height * 2
+
+            Image {
+                id: thumbnailImage
+                width: parent.width * 0.66
+                height: parent.height * 0.66
+                anchors.centerIn: parent
+                cache: true
```

Comment:
Can you make sure that the cache works across restart? Generating a thumbnail is a bit expensive on the server side.

Reply:
Eyeballing it, it seems to work across computer restarts but I'm not sure, is there some way to tell if QML is loading the image from cache/if the server is receiving a GET request for the thumbnail>

Reply:
@CarlSchwan are those previews cached on server side ?

Reply:
so the cache is not kept after a restart
I always see that we download the preview images